### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (kokosing) [inline presto-docs/build:1]: should you store it as `presto-docs/bin/build.sh`?

File: presto-docs/pom.xml [hunk 2]
```diff
@@ -95,6 +95,16 @@
...
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>run-sphinx</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>${project.basedir}/build</executable>
```
Comment (kokosing) [inline presto-docs/pom.xml:105]: don't you need to pass (export) `PRESTO_VERSION`?

**Comment by kokosing** [comment]
> kokosing approved these changes now

I haven't checked the content changes or css files.

File: presto-docs/build (hunk 1 above)
Comment (electrum) [reply presto-docs/build:1]: I considered that, but it seems overkill for a single build script.

File: presto-docs/pom.xml (hunk 2 above)
Comment (electrum) [reply presto-docs/pom.xml:105]: No, that is an override. The Sphinx configuration picks up the version automatically from the POM.

**Pull request merged**